onnxruntime/contrib_ops/cpu/transformers/subgraph_t5_decoder.cc

```diff
@@ -160,9 +160,15 @@ Status T5DecoderSubgraph::CreateInitialFeeds(
   } else {
     for (int i = 0; i < batch_beam_size; i++) {
       gsl::span<const int32_t> sequence = sequences.GetSequence(i);
-      const int32_t* sequence_data = sequence.data();
+      gsl::span<int32_t> seq_span(const_cast<int32_t*>(sequence.data()), sequence.size());
       for (int j = 0; j < cur_len; j++) {
-        input_ids_data[i * cur_len + j] = sequence_data[j];
+        gsl::span<int32_t> temp_input(&input_ids_data[i*cur_len + j] , 1);
```

Comment:
It is too slow to copy token by token. Better to copy one sequence at a time.

Reply:
If we copy the whole sequence then the inference is incorrect, there is a discrepancy in parity between pytorch and ort.

Reply:
Could you try only call host to device copy once (Like prepare a temp buffer in CPU)?